MetaMask/metamask-extension
# feat: Show network badge in detected tokens modal

## **Description**

Adds the network badge to the listing of tokens detected

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28231?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Import an SRP that ERC20s
2. Click "Import tokens" in the banner
3. See network icons in the asset badge

## **Screenshots/Recordings**



### **Before**



### **After**

<img width="356" alt="SCR-20241031-ovkc" src="https://github.com/user-attachments/assets/ba934bad-8e7d-4c46-b47b-273030e81ae2">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
